custom/icds_reports/reports/poshan_progress_dashboard_data.py

```diff
@@ -31,13 +31,22 @@ def calculate_percentage_single_row(row, truncate_out=True):
     return row
 
 
-def calculate_aggregated_row(data, aggregation_level, data_format):
+def calculate_aggregated_row(data, aggregation_level, data_period, unique_id):
```

Comment:
I think following changes we have merged already to master. Why they are coming here again?

Reply:
This branch is not based on master it's based on the setup release which I will deploy to TCL environment to test